SECURITY.md

```diff
@@ -0,0 +1,76 @@
+# OSM Security Process and Policy
+
+This document provides the details on the OSM security policy and details the processes
+surrounding security handling including a how to guide on reporting a security vulnerability
+for anything within the OSM organization.
+
+## Report A Vulnerability
+
+We’re extremely grateful for security researchers and users who report vulnerabilities
+to the OSM community. All reports are thoroughly investigated by a set of OSM maintainers.
+
+To make a report please email the private security list at cncf-openservicemesh-security@lists.cncf.io
+with the details.
+
+You may, but are not required to, encrypt your email to this list using the PGP keys
+of security team members, listed below.
+
+Name               | Key URL                                              | Fingerprint
+------------------ | ---------------------------------------------------- | -----------
+TBD                | TBD                                                  | TBD
+TBD                | TBD                                                  | TBD
+TBD                | TBD                                                  | TBD
+
+
+### When To Send A Report
+
+You think you have found a vulnerability in a OSM project or a dependency of a OSM project. This can be any of the repositories on the [OSM GitHub organization](https://github.com/openservicemesh).
+
+### When Not To Send A Report
+
+* If a vulnerability has been found in an application managed by OSM. Instead, contact the application maintainers
+* You are looking for help applying security updates
+
+### Security Vulnerability Response
+
+Each report will be reviewed and receipt acknowledged within 3 business days. This will set off the security review process detailed below.
+
+Any vulnerability information shared with the security team stays within the OSM project and will not be shared with others unless it is necessary to fix the issue. Information is shared only on a need to know basis.
+
+We ask that vulnerability reporter(s) act in good faith by not disclosing the issue to others. And we strive to act in good faith by acting swiftly, and by justly crediting the vulnerability reporter(s) in writing.
+
+As the security issue moves through triage, identification, and release the reporter of the security vulnerability will be notified. Additional questions about the vulnerability may also be asked of the reporter.
+
+### Public Disclosure
+
+A public disclosure of security vulnerabilities is released alongside release updates or details that fix the vulnerability. We try to fully disclose vulnerabilities once a mitigation strategy is available. Our goal is to perform a release and public disclosure quickly and in a timetable that works well for users. For example, a release may be ready on a Friday but for the sake of users may be delayed to a Monday.
```

Comment:
I didn't get the last part on why a security release could be delayed for users. Could you clarify how delaying a security patch helps?